### PR TITLE
Move Copilot Explain out of experimental and into regular docs

### PR DESCRIPTION
**Pull request by captainbrosset**
The "Copilot Explain" features are no longer experiments and do not need to be enabled. It's time to move the article outside of the experimental features TOC, and into regular docs.

* Before: https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/experimental-features/copilot-explain
* After: 
  * **Explain Console errors and warnings using Copilot in Edge**: [Preview](https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/console/copilot-explain-console?branch=pr-en-us-3112) / [GitHub](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/copilot-explain/microsoft-edge/devtools-guide-chromium/console/copilot-explain-console.md).
  * **Console features reference**: [Preview](https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/console/reference?branch=pr-en-us-3112#explain-console-errors-and-warnings-using-copilot-in-edge) / [GitHub](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/copilot-explain/microsoft-edge/devtools-guide-chromium/console/reference.md#explain-console-errors-and-warnings-using-copilot-in-edge).
  * **Explain source code using Copilot in Edge**: [Preview](https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/javascript/copilot-explain-source-code?branch=pr-en-us-3112) / [GitHub](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/copilot-explain/microsoft-edge/devtools-guide-chromium/javascript/copilot-explain-source-code.md).
  * **JavaScript debugging features**: [Preview](https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/javascript/reference?branch=pr-en-us-3112#explain-source-code-using-copilot-in-edge) / [GitHub](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/copilot-explain/microsoft-edge/devtools-guide-chromium/javascript/reference.md#explain-source-code-using-copilot-in-edge).

AB#49592042

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/console/copilot-explain-console.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+ms.subservice: devtools
+ms.date: 03/20/2024
+---
+# Explain Console errors and warnings using Copilot in Edge
+
+The **Console** tool in Microsoft Edge DevTools integrate with Copilot in Microsoft Edge to help you understand errors and warnings.
+
+When inspecting a webpage with DevTools, you often see errors and warnings in the **Console** tool. These errors and warnings can sometimes be difficult to understand and fix. By using the **Explain this error** feature, you get more information about the error or warning in Copilot in Edge:
+
+![Copilot in the Microsoft Edge sidebar, showing the error message and the explanation.](./copilot-explain-images/copilot-error-explanation.png)
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/console/copilot-explain-console.md:17]:
```suggestion
![Copilot in the Microsoft Edge sidebar, showing the error message and the explanation.](./copilot-explain-console-images/copilot-error-explanation.png)
```

Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/console/copilot-explain-console.md]: fixed

File: microsoft-edge/devtools-guide-chromium/console/reference.md [hunk 2]
```diff
@@ -28,120 +28,147 @@ This article is an outline of the features of the **Console**.
...
 
-<!--
-> ##### Old Figure 9
-> How the logged `XMLHttpRequest` and `Fetch` requests look after ungrouping
-> ![How the logged XMLHttpRequest and Fetch requests look after ungrouping](./reference-images/console-xhr-fetch-all.png)
--->
+1. In the **Console** tool, click the **Console Settings** (![Console Settings icon](./reference-images/settings-button-icon.png)) button, and then elect the **Log XMLHttpRequests** checkbox.
 
-<!--todo: add example for ungrouping console items  -->
+1. In the rendered webpage, click **Get Data** to trigger a `Fetch` API request. The **Console** logs the request and response details:
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/console/reference.md:143]:
```suggestion
1. In the rendered webpage, click the **Get Data** button to trigger a `Fetch` API request. The **Console** logs the request and response details:
```
paint mental picture of the UI controls w/o relying on png or assuming user has right window on screen

**Review by mikehoffms — APPROVED** [review]
Approved PR after pushing commits for a Writer/Editor review

**Review by yanlingwang23 — APPROVED** [review]
LGTM!

**Pull request merged**